inventory/host_vars/zuul.bonnyci-internal.portbleu.com

```diff
@@ -0,0 +1,3 @@
+---
+letsencrypt_csr_cn: zuul.bonnyci.com
```

Comment:
so this is going to be an interesting problem. You can only get a certificate for the public name, because letsencrypt will only issue a cert for something that it can actually contact. 

I'm hoping that with a new cloud and some designate we might be able to do some split private/public DNS-ing that will let us use this internally. 

However the other side is, what things are actually communicating internally via HTTP on the internal network?